vsts/pipelines/nightly.yml

```diff
@@ -61,7 +61,74 @@ jobs:
       echo "##vso[task.setvariable variable=BuildRuntimeImages;]false"
       echo "##vso[task.setvariable variable=TestBuildImages;]false"
       echo "##vso[task.setvariable variable=TestRuntimeImages;]false"
-      echo "##vso[task.setvariable variable=TestIntegrationCaseFilter;]Category!=DB"
+      echo "##vso[task.setvariable variable=TestIntegrationCaseFilter;]Category=PYTHON"
+      echo "##vso[task.setvariable variable=TestIntegration;]true"
+      echo "##vso[task.setvariable variable=PushBuildImages;]false"
+      echo "##vso[task.setvariable variable=PushRuntimeImages;]false"
+      echo "##vso[task.setvariable variable=PushToDockerHub;]false"
+      echo "##vso[task.setvariable variable=EmbedBuildContextInImages;]false"
+    displayName: 'Set variables'
+  - template: _buildParallel.yml
```

Comment:
Instead of running our entire build template, maybe we can just invoke the `test-integration.sh` script?